account/user.go

```diff
@@ -23,8 +23,9 @@ import (
 // User describes a User account. A few identities can be assosiated with one user account
 type User struct {
 	gormsupport.Lifecycle
-	ID                 uuid.UUID          `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
-	Email              string             `sql:"unique_index"`                                            // This is the unique email field
+	ID                 uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	Email              string    `sql:"unique_index"`
+	EmailPrivate       bool
```

Comment:
Since it is a boolean instead of EmailPrivate, IsEmailPrivate would make more sense. But it might involve quite a bit of refactoring, so you can ignore this.

Reply:
I agree with you - but in most places we aren't using the term `is..`  - hence will keep it this way for consistency. :)

Reply:
I don't think we should use "Is" in bool field names at all.